### PR TITLE
Layout "Aus der Neo Welt" added

### PR DESCRIPTION
**Pull request by chrbauer**
See: http://www.adnw.de/

### CODE REVIEW

**Comment by menny** [comment]
thank you. Do you know what is the copyright status of this keyboard? Can it be freely recreated?

**Comment by chrbauer** [comment]
Hm. It is based on neo2 (for all special keys) and it is part of any Xorg installation. It is just hidden, e.g. 
run [gsettings set org.gnome.desktop.input-sources show-all-sources true](http://marcinwolinski.pl/keyboard/) to make it visible.

Unfortunately I couldn't find a copyright notice on adnw.de

There are several implementations, but AFAIK none for android.
http://www.adnw.de/index.php?n=Main.Downloads

**Comment by chrbauer** [comment]
Here is a comment: http://www.adnw.de/index.php?n=Main.Impressum
"All publised data and software are public domain"

**Comment by menny** [comment]
great! thanks.

File: addons/languages/german/pack/src/main/res/xml/de_adnw.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016, kertase, 2020, chrbauer
```
Comment (menny) [inline addons/languages/german/pack/src/main/res/xml/de_adnw.xml:2]: @chrbauer I need to change the license to Apache2. See: https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/CONTRIBUTING.md#license

I would prefer if you submit a new PR that removes the license header.

Thank you!

Comment (chrbauer) [reply addons/languages/german/pack/src/main/res/xml/de_adnw.xml:2]: https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2674

**Pull request merged**